packages/web-components/src/toolbar/toolbar.styles.ts

```diff
@@ -0,0 +1,97 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  fontFamilyBase,
+  spacingHorizontalNone,
+  spacingHorizontalS,
+  spacingHorizontalXL,
+  spacingHorizontalXS,
+} from '../theme/design-tokens.js';
+
+/**
+ * Toolbar styles
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host {
+    --toolbar-item-gap: 4px;
+    align-items: center;
+    background: ${colorNeutralBackground1};
+    border-radius: ${borderRadiusMedium};
+    box-sizing: border-box;
+    color: ${colorNeutralForeground1};
+    font-family: ${fontFamilyBase};
+    height: 40px;
+    padding: ${spacingHorizontalXS} ${spacingHorizontalS};
+    width: 100%;
+  }
+  ::slotted(:not([slot])) {
+    flex: 0 0 auto;
+    flex-direction: row;
+    margin: 0 var(--toolbar-item-gap);
+  }
+
+  .positioning-region {
+    align-items: center;
+    display: inline-flex;
+    flex-flow: row wrap;
+    flex-grow: 1;
+    justify-content: flex-start;
+  }
+  ::slotted([slot='label']) {
+    margin-right: var(--toolbar-item-gap);
```

Comment:
`margin-inline-end`

Reply:
@eljefe223 This is a very significant point! Up until now, I have been applying specific directionality to my padding and margins. Using margin-inline- and margin-block- make MUCH more sense as we support multiple writing and layout directions. Thanks for bringing this up!!